js/ui/text_box/ui.text_editor.base.js

```diff
@@ -447,7 +448,7 @@ const TextEditorBase = Editor.inherit({
 
 
     _cleanLabelObservable: function() {
-        if(this._labelContainerElement) {
+        if(this._labelContainerElement && this._label.isVisible()) {
```

Comment:
Can it really be no label container element?

Reply:
Yes, at the first time we call _cleanLabelObservable early than we define this._labelContainerElement